source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -37,59 +49,102 @@ HTTP/1.1
   </tr>
   <tr>
-    <td class="tg-0pky">vtr</td>
-    <td class="tg-0pky">Optional<br></td>
-    <td class="tg-0pky">The <code>vtr</code> parameter represents ‘Vectors of Trust’. You selected your Vector of Trust when you chose the level of authentication for your service. <br>
-    This parameter is optional. If you do not specify the <code>vtr</code>  parameter, your service will automatically log your users in at the medium level of protection, otherwise known as <code>Cm</code>. There's more <a href=/integrate-with-integration-environment/choose-the-level-of-authentication/#choose-the-level-of-authentication-for-your-service><span> guidance for the level of authentication for your service</span></a> if you need it.</td>
+    <td class="tg-rars"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C">vtr</span></td>
+    <td class="tg-rars"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C">Optional</span></td>
+    <td class="tg-rars"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C">The </span><code>vtr</code><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C"> parameter represents ‘Vectors of Trust’ where you request authentication and, optionally, identity assurance. For example, if you want the medium level of authentication and medium identity confidence, request <code>vtr=[“Cl.Cm.p2”]</code>. </span><br><br><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C">You selected your Vector of Trust when you <a href=https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/choose-the-level-of-authentication><span>chose the level of authentication</span></a> and <a href=https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/choose-the-level-of-identity-confidence><span>the level of identity confidence</span></a> for your service.</span><br><br><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C">You can read more about how to combine the vectors for authentication level and identity confidence in </span><a href=https://datatracker.ietf.org/doc/html/rfc8485#section-3.1><span>Section 3 of RFC 8485</span></a>. If you need identity assurance, you must request <code>Cl.Cm</code> (the medium level of authentication) or higher (higher levels of authentication will be available soon).<br><br><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C">If you do not specify the <code>vtr</code></span><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C"> parameter, your service will automatically log your users in at the medium level of authentication (<code>Cl.Cm</code>).</span><span style="font-weight:400;font-style:normal;text-decoration:none;color:#0B0C0C"> This means you will not receive identity information in your response. </span><br></td>
```

Comment:
We have more missing quote marks from the hrefs here - would be easier to spot with more new lines ;-)